docs/jsonapi-spec.md

```diff
@@ -592,6 +593,87 @@ See [Multi-Document Failure Considerations](#multi-document-failure-consideratio
+
+```json5
+// Find a collection, return complete documents
+{"findCollection" : {} }
+
+// Select where name == "John" and return selected fields
+{"findCollection" : {
+    "namespace" : {"myNamespace"},
+    "projection" : {"???": 1, "???": 1}
+    } 
+} 
+```
+
+#### findCollection Command Order of Operations
+
+TODO: add how cursor state fits into the processing
+
+`findCollection` commands are processed using the following order of operation:
+
+1.  `<namespace>` specifies the namespace to query for all of its valid collection tables. 
+    If no namespace is given, return error (???). If the provided namespace does not exist, return `NAMESPACE_DOES_NOT_EXIST`.
+2.  `<sort-clause>` is applied to the candidate documents to order them,
+    if no `<sort-clause>` is supplied the documents sort order is
+    undefined.
+3.  The `limit` option from `<findCollection-command-options>` is applied to
+    reduce the number of candidate documents to no more than `limit`, if
+    no `limit` is supplied all candidate documents are included.
+4.  `<projection-clause>` is applied to each candidate document to
+    create the **result set** documents to be included in the
+    `<findCollection-command-response>`, if no `<projection-clause>` is specified
+    the complete candidate documents are included in the response.
+
+#### findCollection Command Options
+
+`<findCollection-command-options>` is a map of key-value pairs that modify the behavior of the command. All options are optional, with default behavior
+applied when not provided.
+
+| Option       | Type               | Description                                                     |
+| ------------ | ------------------ | --------------------------------------------------------------- |
+| `limit`      | Positive Integer   | Limits the number of documents to be returned by the command. If unspecified, or 0, there is no limit on number of documents returned. Note that results may still be broken into pages. |
+| `pageState`  | ASCII String       | The page state of the previous page, when supplied the `find` command will return the next page from the result set. If unspecified, null, or an empty string the first page is returned. |
+| `skip`       | Positive Integer   | Skips the specified number of documents, in sort order, before returning any documents. |
+
+#### findCollection Multi Document Failure Modes
+
+Fail Fast, a storage failure causes the command to stop processing.
+
+See [Multi-Document Failure Considerations](#multi-document-failure-considerations).
+
+#### findCollection Command Response
+
+| Response Elements | Description                                                                     |
+| ----------------- | ------------------------------------------------------------------------------- |
+| `data`            | Present with fields : `docs`, `count` and `nextPageState`. `nextPageState` may be `null` if no further data is available. |
+| `status`          | Not preset. |
```

Comment:
status has `collections` field with array of the available collection names

Reply:
fixed in next commit